lib/game.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+class Game
+  def initialize(player1, player2)
+    @player1 = player1
+    @player2 = player2
+    @player1_played = []
+    @player2_played = []
+    @board_array = []
+    @box_played = []
+    @current_player = 1
+    @winning_combos = [[1, 2, 3], [4, 5, 6], [7, 8, 9], [1, 4, 7], [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7]]
+    @ct = 0
+  end
+
+  def board
+    # >>> This creates the board and marks every grid with a number 1 - 9
+    # for i in 1..9
+    #   @board_array.push(" [ #{i} ] ").each_slice(3)
+    # end
+    (1..9).each do |i|
+      @board_array.push(" [ #{i} ] ").each_slice(3)
+    end
+    @board_array.each_slice(3) { |x| puts x.join puts }
+  end
+
+  def play
+    # While nobody has won
+    board
+    while @ct < 9
+      if @current_player == 1
+        puts "Choose a number between 1-9 #{@player1}"
```

Comment:
Let's keep game logic away from user interaction.

> main.rb file should be responsible for user interaction. This is the only one place where puts and get.chomp can be used.

### Consider moving all user interaction to the `main.rb` file.

Reply:
Thank you for this suggestion! We will fix this error as soon as we can. 😄

Reply:
@bolah2009, is it okay to still keep some "puts" in the game class? For example, when the game ends in a win, is it okay to output the winner in the game.rb?

Reply:
I understand that user interaction goes into the main.rb, but I took a closer look at the project outline and read, and I quote, **_"As main.rb file should be responsible for user interaction, this is the only(!) one place where puts/print and get.chomp can be used."_**

Reply:
Hi @yirano, that quote means the `main.rb` file is the only place `puts`/`print` is allowed. 

You can, however, output `String` in this class then `main,rb` can display them (using `puts`/`print`)

Reply:
See [this comment too](https://github.com/maryfelmendoza/tictactoe/pull/3#pullrequestreview-298979017) :blush:

Reply:
Oh wait! I forget that you can return a String. Forgive my mess, my brain has become scrambled with information overload this week. 

Thank you so much for the clarification @bolah2009!

Reply:
:smile: I can understand, it's definitely not a mess. :+1: